src/jailer/src/cgroup.rs

```diff
@@ -72,7 +68,8 @@ impl CgroupBuilder {
         ).map_err(JailerError::RegEx)?;
 
         for l in BufReader::new(f).lines() {
-            let l = l.map_err(|err| JailerError::ReadLine(PathBuf::from(PROC_MOUNTS), err))?;
+            let l =
+                l.map_err(|err| JailerError::ReadLine(PathBuf::from(&b.proc_mounts_path), err))?;
```

Comment:
nitpicking, but & can be removed.
```suggestion
                l.map_err(|err| JailerError::ReadLine(PathBuf::from(b.proc_mounts_path), err))?;
```

Reply:
fixed in https://github.com/firecracker-microvm/firecracker/commit/e77496a63070784ab99d69f438c221555d99b990